leecade/react-native-swiper
# fix internal offset index when passing index props

### Is it a bugfix ?
- Yes
- If yes, which issue (fix #701 ) ?
Yes

### Is it a new feature ?
- No ?
- Include documentation, demo GIF if applicable

### Describe what you've done:

### How to test it ?
When you pass index props, after scroll end or begin, so app crash.
I fixed it by update offset, you can check code
